app/routes/apply/new-eligibility/about-you.js

```diff
@@ -12,30 +12,62 @@ const dateFormatter = require('../../../services/date-formatter')
 const benefitsHelper = require('../../../constants/benefits-enum')
 const NORTHERN_IRELAND = 'Northern Ireland'
 
+const REFERENCE_DOB_ERROR = '?error=expired'
+
 module.exports = function (router) {
   router.get('/apply/:claimType/new-eligibility/:dob/:relationship/:benefit/:referenceId', function (req, res) {
```

Comment:
as above, the point of the change is to remove these params (except claim type) from URL so they don't appear in history